images/php/fpm/Dockerfile

```diff
@@ -18,7 +18,7 @@ ENV LAGOON_VERSION=$LAGOON_VERSION
 
 # Copy commons files
 COPY --from=commons /lagoon /lagoon
-COPY --from=commons /bin/fix-permissions /bin/ep /bin/docker-sleep /bin/
+COPY --from=commons /bin/fix-permissions /bin/ep /bin/docker-sleep /bin/entrypoint-readiness /bin/
```

Comment:
mhh we don't need this in the `php` image? could we add this just to the `php-cli` image?